### PR TITLE
Add enemy movement

### PR DESCRIPTION
**Pull request by henriiquecampos**
Very simple enemy movement.

### CODE REVIEW

File: game/actors/Enemy.gd [hunk 1]
```diff
@@ -0,0 +1,10 @@
+extends "res://actors/PlatformActor.gd"
+
+func _ready() -> void:
+	_linear_velocity.x = -movement_speed
```
Comment (razcore-rad) [inline game/actors/Enemy.gd:4]: How about adding a random direction to give it a little flavor?

```gdscript
func _ready() -> void:
	var rng: = RandomNumberGenerator.new()
	rng.randomize()
	_linear_velocity.x = movement_speed * (-1 if rng.randf() < 0.5 else 1)
```

Also, forgot to mention this in the other PR. Let's rename `movement_speed` to `speed`. Speed is about moving so it's redundant.

**Comment by razcore-rad** [comment]
Other than the above I don't have anything to add, this is good for merging.

File: game/actors/Enemy.gd (hunk 1 above)
Comment (henriiquecampos) [reply game/actors/Enemy.gd:4]: From the reference in #1, Mario's goombas, and most enemies, generally start walking towards left (towards the player).

>A single enemy type like Mario's goombas, that moves forward and turns around when it hits a wall,

I guess is better to maintain the "always going towards the player's" behavior, for design purposes:

Put an enemy next to cliff, by design it should fall to the cliff so the player can jump on it, hop and fall on the cliff's edge, like in some Mario's level. With randomization, this design is broken. Tho, we could use a wall to make it bounce and get in track for this design...

So designing, and simplicity wise I'd rather keep them this way. Tho, I think would be a nice thing to add some randomization, I wonder where, maybe on the score? Like enemies giving a random amount of points within a range, in that sense we could use `rand_range` which I guess is a simpler approach and is most likely already stablished in the viewer's toolkit.

I just read your comment on #3 and it sounds like this mechanic of dropping coins is a good catch. We can go with that to use randomization.

Comment (razcore-rad) [reply game/actors/Enemy.gd:4]: Agreed about the enemy movement. We won't use a random direction

**Comment by razcore-rad** [comment]
We could add random scoring, but I guess we'll talk about it in the respective PR

**Pull request merged**